docs/content/v2024.2/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-setup.md

```diff
@@ -42,10 +42,7 @@ Ensure the universes have the following characteristics:
 
 Prepare your database and tables on the DR primary. The DR primary can be empty or have data. If the DR primary has a lot of data, the DR setup will take longer because the data must be copied in full to the DR replica before on-going asynchronous replication starts.
 
-On the DR replica, create a database with the same name as that on the DR primary.
-
-- During DR setup in [Manual mode](../#manual-schema-changes), you don't need to create objects on the DR replica; DR performs a full copy of the data to be replicated on the DR primary, and automatically creates tables and objects, and restores data on the DR replica from the DR primary.
-- In [Semi-automatic mode](../#semi-automatic-schema-changes), you need to create all objects (tables, indexes, and so on) on the DR replica exactly as they are on the DR primary prior to setting up xCluster DR.
+During DR setup (regardless of mode), you don't need to create objects on the DR replica; DR performs a full copy of the data to be replicated on the DR primary, and automatically creates tables and objects, and restores data on the DR replica from the DR primary.
```

Comment:
@ddhodge : We can't merge these two. We currently have a limitation in semi automatic mode where we need these objects created up front.

Reply:
Not clear what needs to happen. Should line 45 remain
"On the DR replica, create a database with the same name as that on the DR primary. During initial DR setup, you don't need to create objects on the DR replica. DR performs a full copy of the data to be replicated on the DR primary, and automatically creates tables and objects, and restores data on the DR replica from the DR primary."